components/Note/src/index.css

```diff
@@ -0,0 +1,29 @@
+.denhaag-note {
+  align-items: flex-start;
+  color: var(--denhaag-note-color);
+  display: flex;
+  flex-direction: row;
+  font-family: var(--denhaag-note-font-family, inherit);
+  font-size: var(--denhaag-note-font-size, 1.125em);
+  font-weight: var(--denhaag-note-font-weight, normal);
+  line-height: var(--denhaag-note-line-height, 1.3);
+  gap: var(--denhaag-note-icon-spacing);
+  padding-block-start: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+  padding-block-end: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+}
+
+.denhaag-note--info {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-info-color, #ffffff);
+}
+
+.denhaag-note--warning {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-warning-color, #ffffff);
+}
+
+.denhaag-note__icon,
+.denhaag-note svg {
+  aspect-ratio: 1 / 1;
+  flex: 0 0 auto;
+  height: auto;
+  width: var(--denhaag-note-icon-size, calc(var(--denhaag-note-icon-spacing) * 3));
+}
```

Comment:
The `--denhaag-note-icon-color` token is not yet being applied to the svg.

Reply:
I added the token to the svg in commit: https://github.com/nl-design-system/denhaag/pull/611/commits/5181b3b31fff17bdb34c0dab0252adf74b6f965d